model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -88,7 +88,7 @@ class FindingsMatcher(
     /**
      * Associate copyright findings to license findings within a single file.
      */
-    private fun associateFileFindings(
+    private fun matchFileFindings(
```

Comment:
Commit message: For consistency with what?

Reply:
With the `match` function in that same file to which this function is closely related.